roles/ceph-common/defaults/main.yml

```diff
@@ -55,6 +54,8 @@ ceph:
   rbd_default_features: 3
   rbd_default_map_options: rw
   rbd_default_format: 2
+  rbd_url: https://raw.githubusercontent.com/ceph/ceph/v0.94.5/src/pybind/rbd.py
```

Comment:
We might be able to optimize this on master, but this is a cherry pick from what's on master to the 2.0.x branch.